tools/query-audit/auditor.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
```

Comment:
Not sure why this tools/query-audit was added. May be a rebase ?

Reply:
This was added to get the diffs between results when querying sharded and non sharded versions of the read path. The output is included in the _Tooling_ section of the PR description.

Reply:
Sorry I missed this. will take a look